changelog.d/20230425_115025_derek_implement_get_run_sc_23288.rst

```diff
@@ -0,0 +1,3 @@
+
+* `[sc-23288] <https://app.shortcut.com/globus/story/23288>`_
```

Comment:
I would like us to keep links to internal trackers and resources out of the public documentation, including public changelogs. Otherwise, it's confusing for a user reading the changes.

Having a link in the PR and discussion is fine -- just not in the "finished product".

Reply:
Sure, new rev